Library/Homebrew/Gemfile

```diff
@@ -12,51 +14,68 @@ end
 # * nokogiri - use rexml instead for XML parsing
 
 # installed gems (should all be require: false)
-gem "bootsnap", require: false
-gem "byebug", require: false
-gem "json_schemer", require: false
-gem "minitest", require: false
-gem "parallel_tests", require: false
-gem "pry", require: false
-gem "ronn", require: false
-gem "rspec", require: false
-gem "rspec-github", require: false
-gem "rspec-its", require: false
-gem "rspec_junit_formatter", require: false
-gem "rspec-retry", require: false
-gem "rspec-sorbet", require: false
-gem "rubocop", require: false
-gem "rubocop-ast", require: false
-gem "simplecov", require: false
-gem "simplecov-cobertura", require: false
-gem "warning", require: false
-
-group :sorbet, optional: true do
-  gem "parlour", require: false
-  gem "sorbet-static-and-runtime", require: false
-  gem "spoom", require: false
-  gem "tapioca", require: false
+# ALL of these gems that are not vendored should be in a group
+group :ast, optional: true do
```

Comment:
Can these group names match command names instead? I think that'll be easier to follow.

Reply:
They all are more or less except `rspec` (more obvious than `tests` IMO) and this one. `ast` is used in multiple commands, `brew bottle` and `brew bump-revision`, so wasn't sure what else to call it.

Reply:
I'd prefer `tests`, `bottle`. Could also have `bump-revision` with an identical group name to make the point. Agreed on `typecheck`.

Reply:
@Bo98 cool, fine with me as-is then, thanks!